CHANGELOG.md

```diff
@@ -4,6 +4,64 @@ All notable changes to this project will be documented in this file.
+
+## Changed
+- [#581](https://github.com/Automattic/VIP-Coding-Standards/pull/581): AlwaysReturnInFilter: flag abstract methods for manual inspection.
+- [#584](https://github.com/Automattic/VIP-Coding-Standards/pull/584): Performance: more selective sniffing for efficiency.
+- [#586](https://github.com/Automattic/VIP-Coding-Standards/pull/586): IncludingNonPHPFile: various improvements.
+- [#587](https://github.com/Automattic/VIP-Coding-Standards/pull/587): LowExpiryCacheTime: new warning added for manual inspection along with various improvements.
+- [#589](https://github.com/Automattic/VIP-Coding-Standards/pull/589): WPQueryParams: inherit from abstract class and flag for 'exclude' parameter.
+- [#592](https://github.com/Automattic/VIP-Coding-Standards/pull/592): DynamicCalls: various improvements.
+- [#595](https://github.com/Automattic/VIP-Coding-Standards/pull/595): Underscorejs: various improvements.
+- [#620](https://github.com/Automattic/VIP-Coding-Standards/pull/620): Ruleset: silence UnusedVariable from VariableAnalysis to reduce noise.
+- [#624](https://github.com/Automattic/VIP-Coding-Standards/pull/624): ProperEscapingFunction: privatize $escaping_functions property along with various improvements.
+- [#635](https://github.com/Automattic/VIP-Coding-Standards/pull/635): Ruleset: remove WordPress.WP.GlobalVariablesOverride.Prohibited severity declaration.
+- [#638](https://github.com/Automattic/VIP-Coding-Standards/pull/638): IncludingFile: allow custom keywords to reduce false positives.
+- [#639](https://github.com/Automattic/VIP-Coding-Standards/pull/639): RestrictedFunctions: remove site_option group.
+- [#644](https://github.com/Automattic/VIP-Coding-Standards/pull/644): RestrictedFunctions: remove wp_cache_get_multi group.
+- [#645](https://github.com/Automattic/VIP-Coding-Standards/pull/645): Ruleset: silence WordPress.WP.AlternativeFunctions.file_system_read_readfile.
+- [#646](https://github.com/Automattic/VIP-Coding-Standards/pull/646): Ruleset: silence WordPress.WP.AlternativeFunctions.file_system_read_fclose.
+- [#647](https://github.com/Automattic/VIP-Coding-Standards/pull/647): RestrictedFunctions: remove get_super_admins group.
+- [#649](https://github.com/Automattic/VIP-Coding-Standards/pull/649): RestrictedFunctions: downgrade switch_to_blog() to warning and change messaging.
+- [#652](https://github.com/Automattic/VIP-Coding-Standards/pull/652): RestrictedFunctions/RestrictedVariables: remove usermeta related errors.
+
+## Fixed
+- [#444](https://github.com/Automattic/VIP-Coding-Standards/pull/444): ConstantString: account for namespace prefixing.
+- [#581](https://github.com/Automattic/VIP-Coding-Standards/pull/581): AlwaysReturnInFilter: fix runtime failure on abstract methods.
+- [#586](https://github.com/Automattic/VIP-Coding-Standards/pull/586): IncludingNonPHPFile: various bug fixes.
```

Comment:
Since there are many (6) included in this PR, listing all of them seems like it would make for a long-winded entry. However, I'm not opposed to calling out a few of them. Which bug fixes in that PR would you prioritize calling out? I'm leaning towards the below:
- case sensitivity of file extension
- allow for "phar" file extension
- extensions in interpolated strings

Reply:
The "phar" should probably be in the "Changed" entry. I agree on the other for the "Fixed" entry.

Reply:
I'm leaning towards the phar file as in "Added" since it's technically a new feature, no?